zeebe-cluster/values.yaml

```diff
@@ -77,3 +77,35 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+zeebeCfg: |- 
+  [[exporters]]
+  id = "elasticsearch"
+  className = "io.zeebe.exporter.ElasticsearchExporter"
```

Comment:
I think we should start to load the exporters from a defined `jarPath` to avoid classpath issues. For example, by adding `jarPath = "exporters/zeebe-elasticsearch-exporter-0.22.1.jar"` next to the exporters `className`.

Reply:
@saig0 is that already there? I mean `jarPath = ` can I use that already?

Reply:
Yes, it is (since the beginner). We were just too lazy to use it :sweat_smile:

Reply:
@saig0 ok.. but I cannot make this change right now as the `elasticsearch` exporter is not in that new location right? 
What I will do for now is to use your recommended path and in the examples, I will add the `jarPath`

Reply:
:+1: